index.html

```diff
@@ -1,248 +1,330 @@
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Microverse Portfolio Website</title>
-    <link rel="stylesheet" href="styles.css">
-    <link rel="preconnect" href="https://fonts.googleapis.com">
-<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-<link href="https://fonts.googleapis.com/css2?family=Inter&display=swap" rel="stylesheet">
-</head>
-<body>
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
     <header>
-        <nav id="mobile-navbar">
-            <img src="assets/menu.png" alt="">
-        </nav>
+      <nav id="mobile-navbar">
+        <img src="assets/menu.png" alt="" />
+      </nav>
     </header>
     <section id="headline-section">
-        <div class="container">
-            <h1 >
-                <span id="name-heading">
-                    Hey there<br>
-                I'm Abdul Salam
-                </span>
-                <span id="job-role">
-                    I',m a Software<br>
-                    Developer
-                </span>
-                
-            </h1>        
-            <p>
-                I can help you build a product, feature or website Look through some
-                of my work and experience! If you like what you see and have a project
-                you need coded, don’t hesitate to contact me.
-              </p>
-           
-              <div id="social-icons">
-                <ul>
-                    <li><a href="https://twitter.com/Abdul_SalamAbro"><img src="assets/icons/twitter.png" alt=""></a></li>
-                    <li><a href="https://www.linkedin.com/in/abdul-salam-abro/"><img src="assets/icons/linkedin.png" alt=""></a></li>
-                    <li><a href="https://medium.com/abdulsalam"><img src="assets/icons/medium.png" alt=""></a></li>
-                    <li><a href="https://github.com/AbdulSalamAbro"><img src="assets/icons/github.png" alt=""></a></li>
-                    <li><a href="https://venture.angellist.com/v/Abdulsalam"><img src="assets/icons/angellist.png" alt=""></a></li>  
-                </ul>
-            </div>
-    
-            <section id="arrow-down">
-                
-                <a href="#"></a><img src="assets/icons/button_down.png" alt="">
-            </section>
+      <header id="navbar-grid">
+        <nav id="desktop-navbar">
+            <a href="#home">Portfolio</a>
+            <a href="#About">About</a>
+            <a href="#Contact">Contact</a> 
+    </nav>
```

Comment:
- [ ] Great job building the desktop version of your portfolio. I noticed that the nav is displaying nav menu items on the mobile version. THis items are meant to be hidden on the mobile view. Kindly remove it from the mobile view to match template.
### Figma
![image](https://github.com/AbdulSalamAbro/portfoliowebsite/assets/94551484/a878d177-cd00-48b6-a170-b5bf3fce839a)
### Your design
![image](https://github.com/AbdulSalamAbro/portfoliowebsite/assets/94551484/f839df77-5fab-4990-be3a-a50949c1ea83)

Reply:
sorry for that I forgot to hide this from the mobile view I am just updating this now